### PR TITLE
Add help info

### PR DESCRIPTION
**Pull request by debuggy**

### CODE REVIEW

File: src/webportal/src/app/job-submission/utils/constants.js [hunk 1]
```diff
@@ -84,32 +84,23 @@ export const PAI_ENV_VAR = [
...
+  taskRoleContainerSize:
+    'https://openpai.readthedocs.io/en/latest/manual/cluster-user/quick-start.html',
+  hivedSkuType:
+    'https://openpai.readthedocs.io/en/latest/manual/cluster-user/quick-start.html',
+  taskRole:
+    'https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-run-distributed-job.html#taskrole-and-instance',
   parameters:
-    'Parameters are key-value pairs that you could save your frequently used values and reference them in command section by their keys.',
-  secrets: `Secrets are used to store sensitive data. The value will be masked and won't be seen by other users.`,
+    'https://openpai.readthedocs.io/en/latest/manual/cluster-user/quick-start.html',
```
Comment (hzy46) [inline src/webportal/src/app/job-submission/utils/constants.js:94]: This should be `https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-use-advanced-job-settings.html#parameters-and-secrets`

File: src/webportal/src/app/job-submission/utils/constants.js [hunk 2]
```diff
@@ -84,32 +84,23 @@ export const PAI_ENV_VAR = [
...
+  secrets: `https://openpai.readthedocs.io/en/latest/manual/cluster-user/quick-start.html`,
```
Comment (hzy46) [inline src/webportal/src/app/job-submission/utils/constants.js:95]: This should be `https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-use-advanced-job-settings.html#parameters-and-secrets`

**Comment by hzy46** [comment]
image

The alignment seems to have problem. The icon should be in the right-top corner?

**Comment by debuggy** [comment]
> 
> 
> image
> 
> The alignment seems to have problem. The icon should be in the right-top corner?

Updated. Could you help check the link of each info icon?

**Comment by hzy46** [comment]
> > image
> > The alignment seems to have problem. The icon should be in the right-top corner?
> 
> Updated. Could you help check the link of each info icon?

The links for parameter and secrets are wrong. Other links are OK.

**Comment by debuggy** [comment]
> 
> 
> > > image
> > > The alignment seems to have problem. The icon should be in the right-top corner?
> > 
> > 
> > Updated. Could you help check the link of each info icon?
> 
> The links for parameter and secrets are wrong. Other links are OK.

More links are added in constants after this update

**Comment by hzy46** [comment]
Checked. All links are correct now.

**Review by hzy46 — APPROVED** [review]

**Pull request merged**